tune_sklearn/tune_search.py

```diff
@@ -888,6 +918,9 @@ def _fill_config_hyperparam(self, config):
                 configuration for `tune.run`.
 
         """
+        if self.search_optimization == "bayesian":
+            return
```

Comment:
Not sure if this is right. I know you're passing this into `BayesOptSearch`'s `space` parameter which seems correct, but I'm not sure if leaving this out of the `config` dictionary would ruin anything. Just double check to make sure this is good!

Reply:
This seems to be okay based on tune's example; the only parameter passed into config is iterations which doesn't look like a hyperparameter being tuned.

https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/bayesopt_example.py